packages/component-library/stories/Icons.ts

```diff
@@ -69,6 +69,7 @@ import maximize from "../icons/maximize.icon.svg"
 import repeats from "../icons/repeats.icon.svg"
 import zoomIn from "../icons/zoom-in.icon.svg"
 import zoomOut from "../icons/zoom-out.icon.svg"
+import connect from "../icons/connect.icon.svg"
```

Comment:
Brill, thanks for that. The final step is to add the `connect` icon into the `Actions` object located on line 188, which will add it to the [All Icons story](https://cultureamp.design/storybook/?path=/story/components-icon--sticker-sheet-default). Probably best we put it at the bottom of the object too since its a new one 👍

Reply:
Fixed. Sorry missed that line, thanks!